yarn.lock

```diff
@@ -1799,7 +1799,7 @@
   dependencies:
     "@types/markdown-it" "*"
 
-"@types/markdown-it@10.0.2":
+"@types/markdown-it@*", "@types/markdown-it@10.0.2":
```

Comment:
did we want to avoid * imports

Reply:
We want to avoid separated references. Combined ones are okay and work as expected.